### DISCUSSION TITLE
How to calculate multiple plots / threads ?

### DISCUSSION THREAD

**Discussion by grobalt** [madMAx43v3r/chia-plotter#388] [Q&A]
The thread parameter is a multiplier and not a max number.
As i am planning a dual socket 64core setup (multiple of them) .. it would be very very useful to know how many threads are really assigned.
Is there an information how many are used in each Phase or Sub-Phase ?

Thanks a lot !

**Comment by madMAx43v3r** [comment]
Phase 1 has about 3 to 4 times as much, phase 2 => 1.5x, phase 3 => somewhere around 2 to 3 times.

However, here comes the important part, only `-r` number of threads are able to fully max out a CPU core, the rest is usually waiting (partly) on those "main" threads to do their work.

**Reply by grobalt** [reply]
Thanks, i think i did not get the point, sorry. -r 128 on 128 physical core does not seem to be efficient, right ? What happens running 8 instances with -r 16 ? Would that be overprovisioning ?

**Comment by Thelyoncrypt** [comment]
SO if I start the Second plot at half way through phase 2 it should be Ideal?

I am using Raid 0 4 X WD 850 Black 512GB drives with Hyper ASUS 4.0 Extender (TEMP 1)

AMD Thread ripper 25 core 48 Threads

98GB RAM DDR4 3200

I have Seagate Firecuda 520 RAid 0 Temp1 X 2 drives

ASUS Zenith II Extrema Alpha

When is the optimal time to start second plot I am currently getting 1422 Seconds 1 plot if I can increase plot by 50% running 2 that would be fantastic.

Zenith II is a tricky board to get max bandwidth from Raid 0 Optimised raid 0 WD 850 Blacks 2 Drives in the Dimm2 slots and 2 drives in the Asus Hyper NVME extender ( I am about to change raid 0 setup, I have been reading deep in the manual, I found out theoretical speeds can be achieved  15GBS write and 32GBS Read I need to alter this) 

Are you able put some light on manual second plot start? I am going to run 20 plots on each job just hope they don't go out of optimal alignment.

Some good  info from all my testing

NVME 970 Pro/980Pro/WDBLACK850/ ADATA8200 the best drives they don't suffer sustained R?W drops like the others I went from 500MB R/W to 500mb on the Firecudas 520, Horrible drives don't buy them.
